src_c/mixer.c

```diff
@@ -1013,10 +1013,9 @@ static PyTypeObject pgSound_Type = {
     0,             /* tp_dictoffset */
     sound_init,    /* tp_init */
     0,             /* tp_alloc */
-    0,             /* tp_new */
+    PyType_GenericNew,    /* tp_new */
```

Comment:
It's fine if it's a `0` here, `tp_new` is `PyType_GenericNew` by default anyways